flixel/addons/display/FlxTiledSprite.hx

```diff
@@ -106,6 +107,14 @@ class FlxTiledSprite extends FlxStrip
 		return this;
 	}
 
+	override function set_clipRect(Value:FlxRect):FlxRect
+	{
+		if (Value != clipRect)
+			regen = true;
+
+		return super.set_clipRect(Value);
+	}
+
```

Comment:
```suggestion
	override function set_clipRect(value:FlxRect):FlxRect
	{
		regen = true;

		return super.set_clipRect(value);
	}

```

couple things
1. use lower case arg names
2. regen even if the same rect instance is set. one common way to use this is

```hx
sprite.clipRect.width = 100;
sprite.clipRect = sprite.clipRect;
```